docs/graasp-insights-user-guide.md

```diff
@@ -0,0 +1,243 @@
+# Datasets
+
+## Menu navigation
+
+When opening Graasp-Insights, you are greeted with the `Datasets` tab.
+Use the top-left menu button to navigate to each distinctive tab.
+
+![Datasets tab with menu open](./figures/datasets-menu.png)
+
+## Importing a dataset
+
+Load a dataset into Graasp Insights by using the lower right `+` button.
+
+Choose a dataset to import by navigating to its location in your file system.
+You can optionally give it a name and description for its use in Graasp Insights.
+
+![Add dataset form](./figures/add-dataset.png)
+
+Importing a dataset will create a copy of the file and manipulating it through Graasp Insights will have no impact on the original file.
+Only `json`, `csv` and `xlsx` datasets are supported in the current state.
+
+## Dataset management
```

Comment:
Maybe add one line in this section: _If the dataset conforms to an existing schema, it will be tagged with that schema. For more about Schemas see section..._